src/tests/ftest/harness/advanced.py

```diff
@@ -43,24 +45,21 @@ def test_core_files(self):
                 pid = match[0]
             index += 1
         if pid is None:
-            self.fail(
-                "Error obtaining pid of the daos_engine process on "
-                "{}".format(host))
+            self.fail("Error obtaining pid of the daos_engine process on {}".format(host))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string